Description: Fix get_rcvmore on big-endian systems
 This fixes a bug in get_rcvmore on big-endian systems like s390x.
Author: Jan Niehusmann <jan@debian.org>
Forwarded: https://github.com/erickt/rust-zmq/pull/393
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: zmq/src/lib.rs
===================================================================
--- zmq.orig/src/lib.rs
+++ zmq/src/lib.rs
@@ -767,7 +767,7 @@ impl Socket {
 
     /// Return true if there are more frames of a multipart message to receive.
     pub fn get_rcvmore(&self) -> Result<bool> {
-        sockopt::get(self.sock, zmq_sys::ZMQ_RCVMORE as c_int).map(|o: i64| o == 1i64)
+        sockopt::get(self.sock, zmq_sys::ZMQ_RCVMORE as c_int).map(|o: i32| o == 1i32)
     }
 
     sockopts! {
